configurations/aws/i3en_2xlarge.yaml

```diff
@@ -1 +1,5 @@
 instance_type_db: 'i3en.2xlarge'
+
+stress_cmd_w: "cassandra-stress write no-warmup cl=QUORUM duration=600m -schema 'replication(factor=3)' -port jmx=6868 -mode cql3 native -rate 'threads=80 throttle=9000/s' -col 'size=FIXED(128) n=FIXED(8)' -pop 'dist=gauss(1..250000000,125000000,12500000)' "
+stress_cmd_r: "cassandra-stress read no-warmup  cl=QUORUM duration=600m -schema 'replication(factor=3)' -port jmx=6868 -mode cql3 native -rate 'threads=80 throttle=8000/s' -col 'size=FIXED(128) n=FIXED(8)' -pop 'dist=gauss(1..250000000,125000000,12500000)' "
+stress_cmd_m: "cassandra-stress mixed no-warmup cl=QUORUM duration=600m -schema 'replication(factor=3)' -port jmx=6868 -mode cql3 native -rate 'threads=80 throttle=7000/s' -col 'size=FIXED(128) n=FIXED(8)' -pop 'dist=gauss(1..250000000,125000000,12500000)' "
```

Comment:
For reference, can you please add here in the comments the commands that are used for i3.2x?

Reply:
why do we need to repeat it? i think if someone wants to see move info, the way to get to it is from the `.jenkinsfile`, and from there, the 2 `.yaml` files that will contain all the information needed...
but if you insist this is important, i will send a fix for it

Reply:
I meant to show it in the PR comments, not repeating it.

Reply:
added in the description of this PR